.github/workflows/nightly_snapshot.yaml

```diff
@@ -236,11 +236,14 @@ jobs:
         if: ${{ needs.check-for-secrets.outputs.has-openvsx-pat == 'yes' && needs.check-for-secrets.outputs.has-vscode-marketplace-pat == 'yes' }}
         steps:
             - uses: actions/checkout@v4
+            - uses: pnpm/action-setup@v4.0.0
+              with:
+                version: 9.11.0
             - name: Install GNU Sed
               run: brew install gnu-sed
             - uses: actions/setup-node@v4
               with:
-                  node-version: 18
```

Comment:
Maybe we should keep our install-node action, so that we have one central place to set the node-version? We'd need to use that more consistently of course ;-)

Not relevant for this PR.

Reply:
Good point.